Makefile

```diff
@@ -17,3 +17,6 @@ install: build
 vet:
 	go vet .
 	cd S3Proxy && go vet .
+
+test:
+	cd S3Proxy && go test -cover
```

Comment:
Running make test results in the following error:

```bash
cd S3Proxy && go test -cover
go tool: no such tool "cover"; to install:
        go get golang.org/x/tools/cmd/cover
make: *** [test] Error 3
```

Won't you please amend the README